### PR TITLE
Home work 10.10.2020

### PR DESCRIPTION
**Pull request by Asytik13**
- added comments to bugs in code for task1;
- implemented temperature converter;
- added 1 junit test.

### CODE REVIEW

File: .idea/libraries/com_bluecatcode_junit_junit_4_10_extended_1_0_4.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```
Comment (amukas) [inline .idea/libraries/com_bluecatcode_junit_junit_4_10_extended_1_0_4.xml:1]: `.idea` should be in `.gitignore`

some tuts I've found:
 - https://www.youtube.com/watch?v=mf2-MOl0VXY
 - https://www.youtube.com/watch?v=gKWnufyBNQA
 - https://www.youtube.com/watch?v=6vvUevxlOp8

File: HomeWork10102020.iml [hunk 2]
```diff
@@ -4,9 +4,10 @@
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+      <excludeFolder url="file://$MODULE_DIR$/src/com/company" />
```
Comment (amukas) [inline HomeWork10102020.iml:7]: same for `.iml`

File: src/converter/TemperatureConverter.java [hunk 3]
```diff
@@ -15,19 +15,22 @@ static String getTemperatureFromInput() throws IOException {
        return inputByUserTemperature = reader.readLine();
    }
```
Comment (amukas) [inline src/converter/TemperatureConverter.java]: please reformat code

File: src/tests/junit/TemperatureConverterTests.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package tests.junit;
+
+import converter.TemperatureConverter;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TemperatureConverterTests {
+    @Test
+    public void firstTest() throws Exception {
+        TemperatureConverter uut = new TemperatureConverter();
```
Comment (amukas) [inline src/tests/junit/TemperatureConverterTests.java:13]: `sut`? I've never seen `uut`

used widely in https://www.amazon.com/xUnit-Test-Patterns-Refactoring-Code-dp-0131495054/dp/0131495054

Comment (amukas) [reply src/tests/junit/TemperatureConverterTests.java:13]:
```
   @Test
    public void fToC_HappyPath() throws Exception {
        TemperatureConverter sut= new TemperatureConverter();
        String expected = "from F => C -17.77777777777778";

        String actual =  sut.convert("0 F");
        
        assertThat(actual).isEqualTo(expected);
    }
```
compare with yours, I prefer a more pragmatic approach. 

Note: in the assert I propose to use assertJ
https://joel-costigliola.github.io/assertj/

**Review by amukas — APPROVED** [review]
not bad

File: HomeWork10102020.iml (hunk 2 above)
Comment (Asytik13) [reply HomeWork10102020.iml:7]: added to .gitignore

File: .idea/libraries/com_bluecatcode_junit_junit_4_10_extended_1_0_4.xml (hunk 1 above)
Comment (Asytik13) [reply .idea/libraries/com_bluecatcode_junit_junit_4_10_extended_1_0_4.xml:1]: added to .gitignore

File: src/tests/junit/TemperatureConverterTests.java (hunk 4 above)
Comment (Asytik13) [reply src/tests/junit/TemperatureConverterTests.java:13]: changed assertions from jUnit assertions to assertj

**Review by amukas — APPROVED** [review]
LGTM

**Pull request merged**